open-banking-brasil-financial-api-1_ID1.md

```diff
@@ -211,8 +211,8 @@ In addition, the Authorization Server
 1. shall support a signed and encrypted JWE request object passed by value or shall require pushed authorization requests [PAR];
 2. shall distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [OIDD] and [RFC8414]
 3. shall support the claims parameter as defined in clause 5.5 [OpenID Connect Core][OIDC]
-4. shall support the oidc standard claim "cpf" as defined in clause 5.2.2.2 of this document
-5. shall support the oidc standard claim "cnpj" as defined in clause 5.2.2.3 of this document
+4. shall support the oidc standard claim "cpf" as defined in clause 5.2.2.2 of this document if providing access to resources where the resource owner is a `natural person`
```

Comment:
As we always have a person authenticatiing aganst ASPSP system and giving his consent to share data, even when he selected a business account (assuming UX is going to give some option to a user select an especific account), at the end, we always have a CPF that should be related to an account / authentication, right @GislaineAlmeida @alex-siqueira ?

On the other hand, even when a person consents with sharing the data, the owner is a `business` (not a person who allows this sharing)... so, at this point of view, this sentence is not wrong... :-)